### ISSUE TITLE
Installing Vagrant into c:\program files (x86) leads to errors

### ISSUE THREAD

**Issue by tfenster** [hashicorp/vagrant#9880]
### Vagrant version
Vagrant 2.1.1

### Host operating system
Windows 10 [Version 10.0.17134.48]

### Guest operating system
Windows Server 1803 

### Vagrantfile
```ruby
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.require_version ">= 1.6.2"

Vagrant.configure("2") do |config|
    config.vm.define "1803-docker"
    config.vm.box = "1803_docker"
    config.vm.communicator = "winrm"

    # Admin user name and password
    config.winrm.username = "vagrant"
    config.winrm.password = "vagrant"

    config.vm.guest = :windows
    config.windows.halt_timeout = 15

    config.vm.network :forwarded_port, guest: 3389, host: 3389, id: "rdp", auto_correct: true

    config.vm.provider "hyperv" do |v|
        v.cpus = 2
        v.maxmemory = 2048
        v.differencing_disk = true
    end
end

```

### Expected behavior
vagrant box add should run successfully including smb mounts

### Actual behavior
vagrant box add fails with the following error when trying to do the smb mounts
```
Stderr: x86 : The term 'x86' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the
spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:19
+ C:/Program Files (x86)/Vagrant\embedded\gems\2.1.1\g ...
+                   ~~~
    + CategoryInfo          : ObjectNotFound: (x86:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

### Steps to reproduce
1. Install Vagrant in Windows 10 into `c:\Program Files (x86)\Vagrant`
2. Do a vagrant box add of Windows Server 2018 (or other Windows variants, I would guess)
3. Wait for the smb mount 
4. Enter username and password
5. See the failure

### References
Are there any other GitHub issues (open or closed) that should be linked here?
For example:
- https://github.com/laravel/homestead/issues/802#issuecomment-384218481

**Comment by sjroman** [comment]
I'm having the same issue.

### Vagrant version
Vagrant 2.0.4

### Host
Windows 10 Professional Version 10.0.17134 Build 17134

### Guest
Windows 10 Professional Version 10.0.10586 Build 10586


### Vagrantfile
`Vagrant.configure("2") do |config|
  config.vm.guest = :windows
  config.vm.communicator = "winrm"
  config.vm.boot_timeout = 600
  config.vm.graceful_halt_timeout = 600
  config.vm.network :forwarded_port, guest: 3389, host: 3389
  config.vm.network :forwarded_port, guest: 5985, host: 5985, id: "winrm", auto_correct: true
  config.vm.network :forwarded_port, guest: 1521, host: 5555
  config.vm.synced_folder "./test", "/test", type: "smb"
    config.vm.provider "virtualbox" do |vb|
      vb.gui = true
     vb.memory = "4096"
    end
end`

**Comment by chrisroberts** [comment]
This was fixed via #10390 and is available in Vagrant version 2.2.1.

Cheers!
